src/views/ScheduledRestockReview.vue

```diff
@@ -0,0 +1,421 @@
+      <div v-else>
+        <div class="list-item" v-for="(item , index) in parsedItems" :key="index">
+          <ion-item lines="none">
+            <ion-thumbnail>
+              <DxpShopifyImg :src="item.imageUrl" size="small" />
+            </ion-thumbnail>
+            <ion-label>
+              <h2>{{ item.parentProductName }}</h2>
+              <p>{{ item.identification }}</p>
+            </ion-label>
+          </ion-item>
+          <ion-chip outline class="tablet">
+            <ion-label>{{ item.quantity }} incoming</ion-label>
+          </ion-chip>
+          <ion-checkbox :checked="item.isSelected" @ion-change="selectListItem(item)"/>
+        </div>
+      </div>
+      <ion-fab vertical="bottom" horizontal="end" slot="fixed">
+        <ion-fab-button @click="save" :disabled="!parsedItems.length">
+          <ion-icon :icon="cloudUploadOutline" />
+        </ion-fab-button>
+      </ion-fab>
+    </ion-content>
+  </ion-page>
+</template>
+  
+<script>
+import { defineComponent } from "vue";
+import { IonBackButton, IonButton, IonButtons, IonCheckbox, IonChip, IonContent, IonDatetime, IonFab, IonFabButton, IonHeader, IonIcon, IonInput, IonItem, IonLabel, IonList, IonModal, IonPage, IonSearchbar, IonSelect, IonSelectOption, IonThumbnail, IonTitle, IonToolbar, alertController} from "@ionic/vue";
+import { DxpShopifyImg, translate } from "@hotwax/dxp-components";
+import { arrowUndoOutline, checkboxOutline, timerOutline, businessOutline, cloudUploadOutline, globeOutline} from "ionicons/icons"
+import { mapGetters, useStore } from "vuex";
+import { DateTime } from 'luxon';
+import { showToast } from '@/utils';
+import { UploadService } from "@/services/UploadService";
+import { hasError } from "@/adapter";
+import logger from "@/logger";
+import { UtilService } from '@/services/UtilService'
+import emitter from "@/event-bus";
+import { useRouter } from "vue-router";
+
+
+export default defineComponent({
+  name: 'ScheduledRestockDetail',
+  components: {
+    DxpShopifyImg,
+    IonBackButton,
+    IonButton,
+    IonButtons,
+    IonCheckbox,
+    IonChip,
+    IonContent,
+    IonDatetime,
+    IonFab,
+    IonFabButton,
+    IonHeader,
+    IonIcon,
+    IonInput,
+    IonItem,
+    IonLabel,
+    IonList,
+    IonModal,
+    IonPage,
+    IonSearchbar,
+    IonSelect,
+    IonSelectOption,
+    IonThumbnail,
+    IonTitle,
+    IonToolbar
+  },
+  computed: {
+    ...mapGetters({
+      restockItems: 'stock/getRestockItems',
+      facilities: 'util/getFacilities',
+      userProfile: 'user/getUserProfile',
+      fileName: 'order/getFileName',
+      schedule: 'stock/getScheduledInformation',
+      productStores: 'util/getProductStores'
+    })
+  },
+  data() {
+    return {
+      isCsvUploadedSuccessfully: false,
+      originalItems: [],
+      parsedItems: [],
+      isDateTimeModalOpen: false,
+      shopifyShops: []
+    }
+  },
+
+  async ionViewDidEnter() {
+    await this.store.dispatch('util/fetchFacilities');
```

Comment:
I think there is no need to fetch facilities here as we are already fetching them on the first page